example-model/src/main/resources/application.yaml

```diff
@@ -21,26 +21,40 @@ spring:
       enabled: always
   main:
     allow-bean-definition-overriding: true
+  cloud:
+    loadbalancer:
+      ribbon:
+        enabled: false
```

Comment:
I would have thought that you do want ribbon enabled here. Am I wrong?

Reply:
we dont need ribbon enabled here as feign is currently doing all the load balancing

Reply:
OK, I'm not convinced but I'll approve anyway.